lib/puma/dsl.rb

```diff
@@ -759,5 +759,18 @@ def set_remote_address(val=:socket)
     def fork_worker(after_requests=1000)
       @options[:fork_worker] = Integer(after_requests)
     end
+
+    # When enabled, Puma will GC 4 times before forking workers.
+    # If available (Ruby 2.7+), we will also call GC.compact.
+    # Not recommended for non-MRI Rubies.
+    #
+    # Based on the work of Koichi Sasada and Aaron Patterson, this option may
+    # decrease memory utilization of preload-enabled cluster-mode Pumas. It will
+    # also increase time to boot and fork. See your logs for details on how much
+    # time this adds to your boot process. For most apps, it will be less than one
+    # second.
+    def nakayoshi_fork(enabled=false)
```

Comment:
Why are we defaulting this to false? It seems weird that adding this in my config does nothing:

```
nakayoshi_fork
```

I have to 

```
nakayoshi_fork(true)
```

Was this intentional or can we change the default?

Reply:
Thaaaaat would be a bug

Reply:
Maybe that’s why there was so little change in CodeTriage